test/test_iid_random_sequence.py

```diff
@@ -6,13 +6,15 @@
 import pytest
 from scipy.stats import kstest
 
-from pyrenew.metaclass import SampledValue
 from pyrenew.process import IIDRandomSequence, StandardNormalSequence
 from pyrenew.randomvariable import (
     DistributionalVariable,
     StaticDistributionalVariable,
 )
 
+distribution = dist.Cauchy(2, 325.0)
+n = 13532
+
```

Comment:
Should this be here?

Reply:
addressed in https://github.com/CDCgov/PyRenew/pull/441/commits/72af339c87b87be0757d2b79a66f7933bdbb3424